orbit/cmd/orbit/orbit.go

```diff
@@ -251,21 +250,50 @@ func main() {
 		opt.InsecureTransport = c.Bool("insecure")
 
 		var (
-			updater      *update.Updater
 			osquerydPath string
 			desktopPath  string
+			g            run.Group
 		)
 
 		// NOTE: When running in dev-mode, even if `disable-updates` is set,
 		// it fetches osqueryd once as part of initialization.
 		if !c.Bool("disable-updates") || c.Bool("dev-mode") {
-			updater, err = update.New(opt)
+			updater, err := update.New(opt)
 			if err != nil {
 				return fmt.Errorf("create updater: %w", err)
 			}
 			if err := updater.UpdateMetadata(); err != nil {
-				log.Info().Err(err).Msg("update metadata. using saved metadata.")
+				log.Info().Err(err).Msg("update metadata. using saved metadata")
+			}
+
+			targets := []string{"orbit", "osqueryd"}
+			if c.Bool("fleet-desktop") {
+				targets = append(targets, "desktop")
+			}
+			if c.Bool("dev-mode") {
+				targets = targets[1:] // exclude orbit itself on dev-mode.
```

Comment:
nit: This works perfectly fine, but I would change this to be:
```
if !c.Bool("dev-mode") {
   targets = append(targets, "orbit")
}
```
That way we are not dependent on the order of the elements inside targets.

Reply:
Agree, though my reasoning was: I wanted to make "dev" flags be "additional code" vs "modifying production code".